cpp/harvest/marc_issn_lookup.cc

```diff
@@ -43,6 +44,8 @@ struct SubFieldInfo {
     std::string t_;
     std::string w_;
     std::string x_;
+    std::string l_;
```

Comment:
I think that there is no real use in keeping the separate `l_` property.
Moreover, i think we should rename `list_of_l_` to `issns_`.

Reply:
I think the main problem is that SubFieldInfo has become a hybrid struct which is used for 2 different purposes:
1) To represent the data we store in the cache
2) As an intermediate data structure where information about a record is stored so we can share parts of the parsing logic with 1+

Should we split that up into some kind of hierarchy? Let's discuss!